community-operators/jenkins-operator/0.3.31/jenkins-operator.v0.3.31.clusterserviceversion.yaml

```diff
@@ -241,6 +255,7 @@ spec:
             - roles
             - rolebindings
           verbs:
+            - get
```

Comment:
I was thinking that the create/update was missing on this one. Was this PR fixing something?